test/unit/utils/dataProvider.js

```diff
@@ -92,6 +92,7 @@ module.exports = {
     CaseEvent: getCaseEventDefinitions([
       'CaseEvent',
       'CaseEvent-bailiff-nonprod',
+      'CaseEvent-bailiff-or-resp-journey-nonprod',
```

Comment:
are you sure this is meant to be in a new feature toggle?
@adamjhc

Reply:
I had a similar story before and had to merge my work into "share-a-case".
It may be more appropriate to add this into the bailif-nonprod/prod files. However i would check with Jeremy when he plans on releasing this

Reply:
Confirmed with Patryk and Jeremy. This is to be released with whichever release goes ahead first

https://tools.hmcts.net/jira/browse/DIV-6846?focusedCommentId=991307&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-991307

Reply:
We need to do it this way as we have new requirement for event `serviceApplicationReceived` that used to be feature toggled in RRJ, but the requirement is for Bailiff. 

What is more we don't know what will be released first.

Preparing releases is a manual process so we will sort this out when release are being prepared.